rc/windowing/repl/kitty.kak

```diff
@@ -0,0 +1,39 @@
+hook global ModuleLoaded kitty %{
+    require-module kitty-repl
+}
+
+provide-module kitty-repl %{
+
+define-command kitty-repl -params .. -shell-completion -docstring '
+kitty-repl [<arguments>]: create a new window for repl interaction
+All optional parameters are forwarded to the new window' \
+%{
+    nop %sh{
+        if [ $# -eq 0 ]; then
+            cmd="${SHELL:-/bin/sh}"
+        else
+            cmd="$*"
+        fi
+        kitty @ new-window --no-response --window-type $kak_opt_kitty_window_type --title kak_repl_window --cwd "$PWD" $cmd < /dev/null > /dev/null 2>&1 &
+    }
+}
+
+define-command -hidden -params 0..1 \
+    -docstring %{kitty-send-text [text]: Send text to the REPL window.
+
+If no text is passed, the selection is used.} \
```

Comment:
Could you please prettify this docstring, as well as the one for `:kitty-repl`? Something like:

```
-docstring %{
    kitty-send-text [text]: Send text to the REPL window.

    If no text is passed, the selection is used.
} \
```

Reply:
done.